awx/ui/src/screens/Template/JobTemplateDetail/JobTemplateDetail.js

```diff
@@ -167,6 +167,11 @@ function JobTemplateDetail({ template }) {
     );
   };
 
+  const buildLinkURL = (instance) =>
```

Comment:
I think my preference would be to just build the whole path here instead of building some of it here, and the rest in the jsx.  Sort of like we do with `detailsURL` in other places.

Reply:
We already have this function in other parts of the code. Lets keep consistent.